client/__tests__/e2e/cellxgeneActions.js

```diff
@@ -57,9 +57,15 @@ export async function getAllHistograms(testclass, testIds) {
 
   const allHistograms = await getAllByClass(testclass);
 
-  return allHistograms.map((hist) =>
-    hist.dataset.testid.replace(/^histogram-/, "")
+  const testIDs = await Promise.all(
+    allHistograms.map((hist) => {
+      return page.evaluate((elem) => {
+        return elem.dataset.testid;
+      }, hist);
+    })
   );
+
+  return testIDs.map((id) => id.replace(/^histogram-/, ""));
```

Comment:
Non-anno smoke-tests were failing because `getAllHistograms()` was not fetching properly due to `getAllByClass()` rework.

@tihuan Could I get a quick check that my async/await practices are all in order?

Reply:
ah interesting! Yeah thanks for flagging, I'll take a look in 20 mins!

Reply:
Hmm do you know which particular test failed? I just tried `bulk add genes` and it seems to be working!

Reply:
Oops sorry, bad wording. This was my fix for the improper fetch!

Reply:
Just wanted to make sure the async/await syntax in the fix is up to par

Reply:
Oh!! I see 😆  Great work! Yeah the only `await` we need is for `Promise.all()`, because we're transforming `allHistograms` to an array of promises created by `page.evaluate()`. So no further await is needed 👌 

Looks perfect to me 🎉 👍